extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/WebsocketSubProtocolsBuildItem.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.vertx.http.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class WebsocketSubProtocolsBuildItem extends SimpleBuildItem {
```

Comment:
This needs to be a MultiBuildItem, so different extensions can register different sub protocols. Otherwise if two extensions attempt to register sub protocols it will blow up.